connectors/src/connectors/confluence/temporal/client.ts

```diff
@@ -147,3 +153,37 @@ export async function stopConfluenceSyncWorkflow(
     return new Err(e as Error);
   }
 }
+
+export async function launchConfluencePersonalDataReportingSchedule() {
+  const client = await getTemporalClient();
+
+  const action: ScheduleOptionsAction = {
+    type: "startWorkflow",
+    workflowType: confluencePersonalDataReportingWorkflow,
+    args: [],
+    taskQueue: QUEUE_NAME,
+  };
+
+  try {
+    await client.schedule.create({
+      action,
+      scheduleId: makeConfluencePersonalDataWorkflowId(),
+      policies: {
+        // If Temporal Server is down or unavailable at the time when a Schedule should take an Action.
+        // Backfill scheduled action up to 1 previous day.
+        catchupWindow: "1 day",
+        // Only one workflow at a time.
+        overlap: ScheduleOverlapPolicy.SKIP,
+      },
+      spec: {
+        // According to Atlassian's documentation, the cycle period is 7 days.
+        intervals: [{ every: "7d" }],
```

Comment:
No need to action here but we'll likely need a production check for all long running confluence workflows?

Reply:
Yes, it's tracked here: https://github.com/dust-tt/dust/issues/3381.